### PR TITLE
7.1.0

### PR DESCRIPTION
**Pull request by RuiSantosVB**
Add Android migration guide for 7.1.0
Add Android release notes for 7.1.0
Add form feature page

### CODE REVIEW

File: docs/Features/Form/Form_Index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    ```
+
+=== "iOS"
+
+    
+
+The FormError has the following structure:
+
+=== "Android"
+    ```kotlin
```
Comment (JoseRosarioVB) [inline docs/Features/Form/Form_Index.md:163]: Add empty line for formatting reasons (I don't think it works as expected without it)

Comment (JoseRosarioVB) [reply docs/Features/Form/Form_Index.md:163]: Check other examples in this file

File: docs/ReleaseNotes/ReleaseNotes_Android.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
 # MobileID SDK - Release Notes
+## 7.1.0
+- Added improved security measures
```
Comment (JoseRosarioVB) [inline docs/ReleaseNotes/ReleaseNotes_Android.md]: How about changing to "Improved security measures" 
Maybe specify some details regarding hash and hybrid encryption

**Review by vbmobile — APPROVED** [review]

File: docs/Features/Form/Form_Index.md (hunk 1 above)
Comment (RuiSantosVB) [reply docs/Features/Form/Form_Index.md:163]: Looks like this to me, I think it's fine.

Screenshot 2024-03-20 at 10 12 14

Comment (RuiSantosVB) [reply docs/Features/Form/Form_Index.md:163]: With the added line looks exactly the same. We can add it just to make sure its the same everywhere.

**Review by vbmobile — APPROVED** [review]

**Review by JoseRosarioVB — APPROVED** [review]

**Pull request merged**